Running in wrapper mode for '268'

Running python job.

[name: "CPU-QUOTA"
int_val: 16
int_val: 9
int_val: 28
int_val: 47
int_val: 93
int_val: 63
int_val: 22
, name: "DISK"
int_val: 22
int_val: 34
int_val: 7
int_val: 31
int_val: 6
int_val: 27
int_val: 15
, name: "MEMORY"
int_val: 35
int_val: 30
int_val: 25
int_val: 9
int_val: 27
int_val: 39
int_val: 35
, name: "NET"
int_val: 9
int_val: 23
int_val: 30
int_val: 36
int_val: 10
int_val: 28
int_val: 39
]
The paramters are {u'CPU-QUOTA': array([16,  9, 28, 47, 93, 63, 22]), u'NET': array([ 9, 23, 30, 36, 10, 28, 39]), u'DISK': array([22, 34,  7, 31,  6, 27, 15]), u'MEMORY': array([35, 30, 25,  9, 27, 39, 35])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 22500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 117760.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 28521267.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 1125000000.0
Recovering the memory utilization
Setting cpu quota to 40000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 46080.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 18454937.6
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 1312500000.0
Recovering the memory utilization
Setting cpu quota to 70000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 153600.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 5872025.6
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 937500000.0
Recovering the memory utilization
Setting cpu quota to 40000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 46080.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 18454937.6
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 1312500000.0
Recovering the memory utilization
Setting cpu quota to 70000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 153600.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 5872025.6
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 937500000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f403848c878>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.53 seconds.
